src/scripts/main.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  const promise = new Promise((resolve) => {
+    element.addEventListener(eventName, (e) => {
+      resolve(
+        `It was ${eventName} on the element: /${element.nodeName}, id: ${element.id}`,
```

Comment:
The message string in the `resolve` function contains a slash before `${element.nodeName}` which seems unintended. It should likely be removed to correctly format the message.